src/board_filter.py

```diff
@@ -0,0 +1,78 @@
+"""
+This script is intended to be used with the related filter_finder.py. It filters out all the pixels corresponding to a single skateboard in a given controlled video of a skateboard under static conditions.
+
+TODO: make this script configurable to run from the commandline with no need to edit code, e.g. through `python board_filter -v "board.mp4" -c "wheels"`
+
+This script was created as part of a school project, and therefore license-free.
+"""
+
+import os.path
+import cv2
+import numpy as np
+
+__author__ = "Nick"
+
+
+#Increments by which values can be changed through keyboard controls
+sinc = 1
+binc = 5
+
+path = 'We have to manually enter the path to the video file here until we add support for commandline parameters.' #TODO: make path passable from the commandline
```

Comment:
what I like to do is have a separate .py for testing and including specific testing paths (which is not committed), 
that way you don't have this mess.